src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
```

Comment:
The `font-size` and `line-height` are set to 12px and 14px respectively, which might be too small for readability on some devices. Consider using relative units like `em` or `rem` for better scalability and accessibility.